api/mod/src/database/database.py

```diff
@@ -10,7 +10,7 @@
 
 Base = declarative_base()
 
-engine = create_engine(SETTINGS.database_url, connect_args={"check_same_thread": False})
```

Comment:
This option is incompatible with postgres. I tested around a bit and it doesnt seem to break anything locally. We might need to keep an eye on that